requirements/constraints.txt

```diff
@@ -90,3 +90,7 @@ babel==2.11.0
 social-auth-app-django==5.0.0
 algoliasearch==2.6.3
 django-ipware==4.0.2
+
+# pytz>2022 has major changes which are causing test failures.
+# Pinning this version for now so this could be fixed in a separate PR later on
+pytz<2023
```

Comment:
Could you ensure that there's an issue for these failures (either in pytz if they're bugs, or one of our projects if we just need to upgrade things) and add a link to it here?

Reply:
Created followup issue https://github.com/openedx/edx-platform/issues/32093 for arbi-bom team to remove this constraint.